SDK/MobileCenterPush/Microsoft.Azure.Mobile.Push.Android.Bindings/Additions/AboutAdditions.txt

```diff
@@ -0,0 +1,48 @@
+﻿Additions allow you to add arbitrary C# to the generated classes
```

Comment:
We haven't deleted these files in the other modules though. Also, it could be helpful to have them in the future